products/BellHybrid/apps/application-bell-relaxation/ApplicationBellRelaxation.cpp

```diff
@@ -141,6 +156,20 @@ namespace app
             return retMsg;
         }
 
+        if (auto msg = dynamic_cast<db::NotificationMessage *>(msgl); msg != nullptr) {
+            if (msg->interface == db::Interface::Name::MultimediaFiles && msg->type != db::Query::Type::Read) {
+                if (!relaxationRebuildTimerHandle.isActive()) {
+                    relaxationRebuildTimerHandle.start();
+                }
+                //                else {
+                //                    relaxationRebuildTimerHandle.restart(relaxationRebuildTimerInterval);
+                //                }
```

Comment:
Remove this commented code ;)

Reply:
"(jeden fragment kodu jest zakomentowany, bo czekam na opinie jak chcemy odswiezac pliki)"
it will be done after we will confirm which way we want to go :smile: